tests/test_api_client.py

```diff
@@ -120,6 +120,7 @@
 
 CSV_EXPORT = b"meta_file_name,Invoice number\r\nfilename_1.pdf,11111\r\nfilename_2.pdf,22222"
 FAKE_TOKEN = "fake-token"
+OUR_TOKEN = "our-token"
```

Comment:
I'm thinking that renaming the constant to smt like `NEW_TOKEN` would make the added tests more readable.